linux/just_files/just_functions.bsh

```diff
@@ -299,10 +297,9 @@ function source_environment_files()
 
   export JUST_SETTINGS
 
-  local JUST_LOCAL_SETTINGS
-  local JUST_LOCAL_SETTINGS_POST
-  # Sets JUST_LOCAL_SETTINGS and JUST_LOCAL_SETTINGS_POST
-  try_to_create_local_settings_file "${1}" || : # <== Fix for #84
+  local JUST_LOCAL_SETTINGS="${JUST_LOCAL_SETTINGS-}"
+  local JUST_LOCAL_SETTINGS_POST="${JUST_LOCAL_SETTINGS_POST-}"
```

Comment:
What happened to the default values here?

Reply:
They are set in create_local_settings_file